Updated article example text (#25600)
mdn/content: files/en-us/web/css/animation-fill-mode/index.md

### Setting fill mode

> You can see the effect of `animation-fill-mode` in the following example. It demonstrates how, for an animation that runs for specified time, you can cause it to remain in its final state rather than reverting to the original state (which is the default).

Reviewer: ```suggestion
You can see the effect of `animation-fill-mode` in the following example. It demonstrates how you can make the animation remain in its final state rather than reverting to the original state (which is the default).
```
